reference/auth/models/consent/metadata.v1.yaml

```diff
@@ -0,0 +1,22 @@
+title: metadata.v1
+x-stoplight:
+  id: 8ctwq6xjg496g
+type: object
+properties:
+  supportedOrganizations:
```

Comment:
Do we envision other types of consents having attached metadata?

If so, consider making this specific to TBDDP, and perhaps define a top-level `metadata` that uses it as `oneOf`, keyed on `type`.